src/applications/Utilities/src/Utilities.cxx

```diff
@@ -255,6 +255,8 @@ int algorithmsRunner()
   }
   else if (requestedAlgorithm == Nifti2Dicom)
   {
+    std::cout << "!!! WARNING: " <<
+      "Trying to write DICOM from NIfTI is dependent on the fact that the reference DICOM and NIfTI are in the same physical space and describe the same organ type.\n";
```

Comment:
Should the message not be more explicit?

Reply:
I was envisioning something a bit generic. We can make this more verbose/specific as needed in the future.

Reply:
It must say somewhere that the reference dicom should match with nifti. Without that the output is prone to error.

Reply:
Actually, I would counter that **match with nifti** is prone to erroneous interpretation but **the same physical space and describe the same organ type** is not.

Reply:
"same dicom that produced the nifti" is more important than organ type. If this is met, physical dimension condition is automatically met. Same organ type can be t1, t2 or can be a completely different modality which will still produce erroneous result. I leave the wording to you.